packages/animations/lib/src/open_container.dart

```diff
@@ -362,12 +386,40 @@ class _OpenContainerRoute extends ModalRoute<void> {
 
   @override
   final Duration transitionDuration;
+  final ContainerTransitionType transitionType;
 
   final Tween<double> _elevationTween;
   final ShapeBorderTween _shapeTween;
   final _FlippableTweenSequence<Color> _colorTween;
 
-  final _FlippableTweenSequence<double> _closedOpacityTween =
+  // Fade transition opacity tweens
+  static final _FlippableTweenSequence<double> _fadeOpenOpacityTween =
```

Comment:
nit: can you switch open and close tween around so it has the same order ad the fadeThrough tweens?

Reply:
Done